src/static/css/page.css

```diff
@@ -133,6 +133,7 @@
 }
 .content > section, .content > article {
   margin-bottom: 32px;
+  margin-left: 10px;
```

Comment:
What's the purpose of this style change?

Reply:
Thought it looked a little close to the edge without it:

![image](https://user-images.githubusercontent.com/10931297/79670425-d1ae9080-81ba-11ea-8b77-1baa85c88a1b.png)

So this adds a little breathing space:

![image](https://user-images.githubusercontent.com/10931297/79670437-f0148c00-81ba-11ea-8b7c-bfb57501589b.png)

Reply:
Appreciate this does move the article in 10px but think a small price to pay for this useful functionality.

Reply:
Moved this to a better location (which also means it doesn't affect mobile) in https://github.com/HTTPArchive/almanac.httparchive.org/pull/737/commits/a947f817ee11bd3db15aa75abeaa750586ebe466